mkdocs.yml

```diff
@@ -1,10 +1,14 @@
 site_name: Automation for OpenShift on IBM Power 
```

Comment:
@sudeeshjohn can we change the site_name to "Deploying OpenShift on IBM Power Systems"